Scripts/ios-build-release-check.sh

```
@@ -2,13 +2,51 @@
+    return 1
+  fi
+  if printf '%s\n' "$production" | grep -E '^[[:space:]]*xcrun simctl bootstatus'; then
+    echo "Fehler: simctl bootstatus darf im Release-Check-Produktionspfad nicht vorkommen." >&2
+    return 1
+  fi
+  # Parallel Store + Private builds (background jobs + wait) + per-scheme logs.
+  grep -Fq 'build_scheme_release ReiseniOS' "$script"
+  grep -Fq 'build_scheme_release ReiseniOSPrivate' "$script"
+  grep -Fq 'store_pid=$!' "$script"
+  grep -Fq 'private_pid=$!' "$script"
+  grep -Fq 'wait "$store_pid"' "$script"
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)